README.md

```diff
@@ -4,10 +4,12 @@
 
 You're working on your Tunr app and you encounter this error. What does it mean and where would you go first to address it?  
 
-![Rails error](http://i.imgur.com/9NR7XNT.png)  
+![Rails error](http://i.imgur.com/9NR7XNT.png)
 
 ```text
-Your answer...
+andrewkim does not have an `index.html.erb` file where one should reside in `app/views`.  This error is likely to have happened by something like trying to browse to localhost:3000/artists after manually setting up routes and a controller method `index` found in `app/controllers/artists_controler.rb` (don't think it wouldn't have gone that far otherwise).
+
+I'm a bit fuzzy on where the source file might be coming from when the error was raised, and why find is being called in such a way.  I'm guessing it's some sort of rails wrapper of ActiveRecord.
 ```
```

Comment:
@joe-gz  I was wondering what file the source code shown in the error is coming from (where was the error raised?).  More specifically, why was the `MissingTemplate` error raised because the `find` method did not return a `first` index of `find_all`, and why `find` was just a rapper of `find_all`, but only grabbing the first element.  Without context, doing that seems inefficient.

I'm assuming it's a Rails wrapper of ActiveRecord since it's raising a Rails specific error, I guess I'm just curious as to the rails internals, but don't have the time to try to duplicate this message and figure it out.  Digging into the Rails source is something for another time.